expat/lib/expat_external.h

```diff
@@ -93,7 +93,11 @@
 # endif
 #endif  /* not defined XML_STATIC */
 
-#if !defined(XMLIMPORT) && defined(__GNUC__) && (__GNUC__ >= 4)
+#ifndef XML_ENABLE_VISIBILITY
+# define XML_ENABLE_VISIBILITY 0
+#endif
```

Comment:
I wonder if we need that block in neither or both of
- `expat/lib/expat_external.h`
- `expat/lib/internal.h`

Reply:
Right, I updated `internal.h`.